vendor/modules.txt

```diff
@@ -212,17 +212,16 @@ gopkg.in/tomb.v1
 # gopkg.in/yaml.v2 v2.2.4
 gopkg.in/yaml.v2
 # k8s.io/api v0.0.0 => k8s.io/api v0.0.0-20190918195907-bd6ac527cfd2
-k8s.io/api/core/v1
 k8s.io/api/admissionregistration/v1beta1
 k8s.io/api/apps/v1
-k8s.io/api/autoscaling/v1
 k8s.io/api/apps/v1beta1
 k8s.io/api/apps/v1beta2
 k8s.io/api/auditregistration/v1alpha1
 k8s.io/api/authentication/v1
 k8s.io/api/authentication/v1beta1
 k8s.io/api/authorization/v1
 k8s.io/api/authorization/v1beta1
+k8s.io/api/autoscaling/v1
```

Comment:
Please revert all the line reorder, for some reason some local machine reorder,
just keep the real updated lines

Reply:
So I should only add the line in go.mod and do not attempt running `make check`, right?

Reply:
I am used to this from kubevirt, there i do run the `make` targets that are needed, in order to find other changes in case there are, but revert the reordering, this what i would suggest to do here as well imo
best ask Miguel

Reply:
Why is this file important to be reviewed. It is just generated. And how is the order important?

Reply:
it will cause conflicts for others, if anybody would change it as his own machine reorder,
some local machines does changes to the order while some doesn't.

Reply:
Understood. There are no others.

Reply:
If `make format` yield different results every time, it kinda breaks the point of having automated formatter, doesn't it.
I would imagine that we leave the formatting on the tool.

Reply:
You're right Radim. And if we get different results, we should invest in fixing of the tools we use.

I don't see a reason to do anything special about the order in this PR. I think it is good to go once you update commit message and description of the PR as Or suggested in https://github.com/kubevirt/macvtap-cni/pull/25#issuecomment-688752807

Reply:
until we fix it , we will see this file changes again and again imo

Reply:
It is a generated file and there is no need to look at it during reviews.

Reply:
> It is a generated file and there is no need to look at it during reviews.

If a generated file got changed, I interpret it as something was changed that relates to it. It may cause me to check what it is and if that change is correct or not.
Asking the reviewer to ignore changes of any file is a risk of missing unwanted/unintended changes.
Moreover, if this changes every time one builds the project locally and the file is not ignored by git, it requires the developer to drop the changes manually in order to move between branches.

If this file is changing randomly, do not add it to the tracked files or fix the build to reflect nothing changed.

Obviously, this is a general note, it is not a must to block this change on this, but it will be a good idea to fix this sooner then later.

p.s: I do not like generated code being tracked, it seems to exist all over in k8s related projects. I heard no convincing reasoning for it so far. But it is too generic to discuss it on any specific PR, sorry for the noise.

Reply:
in the past when i had it, i found an issue on github about the ordering, worth to try to find it again whoever try to fix this imo.